File: 백준/Bronze/3058. 짝수를 찾아라/짝수를 찾아라.py
t = int(input())

for _ in range(t):
    numbers = list(map(int, input().split()))
    
    total = 0
    minimum = 101

    for number in numbers:
        if number % 2 == 0:
            total += number
            minimum = min(minimum, number)
    
    print(total, minimum)

File: 백준/Bronze/3058. 짝수를 찾아라/짝수를 찾아라.cc
#include <iostream>
using namespace std;

//*	problem 3058
int main(void)
{
	int T;
	cin >> T;

	for (int i = 0; i < T; i++)
	{
		int number;
		int total = 0;
		int minimum = 1000;

		for (int j = 0; j < 7; j++)
		{
			cin >> number;
			if (number % 2 == 0)
			{
				total += number;
				minimum = minimum < number ? minimum : number;
			}
		}
		cout << total << " " << minimum << "\n";
	}

	return 0;
}